io.ente.auth.yml

```diff
@@ -0,0 +1,75 @@
+app-id: io.ente.auth
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: ente_auth
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  # Flutter needs GPU permission.
+  - --device=dri
+  # Needs to talk to the network for authentication
+  - --share=network
+  # For desktop notifications
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=xdg-run/keyring
```

Comment:
why is xdg-run/keyring needed ?

Reply:
I thought libsecret needs it. But I rechecked and seems like it is not required.